pkg/spdk/disk.go

```diff
@@ -266,15 +268,23 @@ func addBlockDevice(spdkClient *SPDKClient, diskName, diskPath string, blockSize
 			return lvstore.UUID, nil
 		}
 
-		log.Infof("Renaming the existing lvstore %v to %v", lvstore.Name, lvstoreName)
-		renamed, err := spdkClient.BdevLvolRenameLvstore(lvstore.Name, lvstoreName)
-		if err != nil {
-			return "", errors.Wrapf(err, "failed to rename lvstore from %v to %v", lvstore.Name, lvstoreName)
-		}
-		if !renamed {
-			return "", fmt.Errorf("failed to rename lvstore from %v to %v", lvstore.Name, lvstoreName)
+		// Rename the existing lvstore to the name of the aio bdev
+		if diskUUID == lvstore.UUID {
+			log.Infof("Renaming the existing lvstore %v to %v", lvstore.Name, lvstoreName)
+			renamed, err := spdkClient.BdevLvolRenameLvstore(lvstore.Name, lvstoreName)
```

Comment:
```
if lvstore.Name != lvstoreName {
    renamed, err := spdkClient.BdevLvolRenameLvstore(lvstore.Name, lvstoreName)
    ......
}
```

Reply:
Sorry. The comment is wrong.
The idea here is only renaming the lvstore which has an expected uuid.

Reply:
Updated the comment, thank you.